packages/citation-js-utils/src/index.ts

```diff
@@ -171,12 +163,26 @@ export function firstNonDoiUrl(str?: string, doi?: string) {
   return matches.map((match) => match[0]).find((match) => !doi || !match.includes(doi));
 }
 
-export async function getCitations(bibtex: string): Promise<CitationRenderer> {
-  const cite = new Cite();
-  const p = await Cite.async(bibtex);
+function parseCitationStyle(style: string): string {
+  const [styleType, styleFormat] = style.split('-');
+  if (styleType !== 'citation') {
+    throw new Error(`unexpected citation style: ${style}`);
+  }
+  return styleFormat;
+}
 
+export async function parseBibTeX(source: string): Promise<CSL[]> {
+  return (await Cite.async(source)).data;
+}
+
+export async function parseCSLJSON(source: object[]): Promise<CSL[]> {
+  return Promise.resolve(cleanCSL(source));
```

Comment:
This doesn't validate the CSL, but does try to "clean" it. Should we also use `Cite.async` here too?